### PR TITLE
Added --user-agent switch, fixed _fetch_user

### PR DESCRIPTION
**Pull request by isync**
follow up to issue #26

### CODE REVIEW

File: bin/youtube-download [hunk 1]
```diff
@@ -187,6 +193,7 @@ Options:
     -h, --help          Display help
     -m, --man           Display man page
     -V, --version       Display version
+    --dump-user-agent   Set this flag to dump the curently used ua-string
```
Comment (oalders) [inline bin/youtube-download]: This isn't actually used anymore, I think.

**Comment by isync** [comment]
leftover from stripping --user-agent from --dump-user-agent

**Comment by oalders** [comment]
This broke the build.  I've triggered a new build to see if it was an actual problem.

File: lib/WWW/YouTube/Download.pm [hunk 2]
```diff
@@ -168,7 +168,7 @@ sub _fetch_title {
 sub _fetch_user {
     my ($self, $content) = @_;
 
-	if( $content =~ /<span class="yt-user-name [^>]+>([^<]+)<\/span>/ ){
+	if( $content =~ /","author":"([^"]+)","/ ){
```
Comment (oalders) [inline lib/WWW/YouTube/Download.pm]: It looks like this change broke `https://github.com/xaicron/p5-www-youtube-download/blob/master/t/video_user.t#L14`

**Comment by isync** [comment]
Sorry for this really rushed patching. The last commit resolves the test issue by supporting older and newer YouTube markup.

**Comment by oalders** [comment]
Thank you!  I squashed everything into one commit and merged it manually.  :)

**Pull request closed without merging**